test/test_torch.py

```diff
@@ -8609,6 +8609,29 @@ def test_slice(self):
         self.assertEqual(x[:, -2:3].tolist(), [[2], [6], [10], [14]])
         self.assertEqual(x[0:-1:2].tolist(), [[0, 1, 2, 3], [8, 9, 10, 11]])
 
+    def test_split_with_sizes_copy_out(self):
```

Comment:
We should have an OpInfo-based test for this that already exists.
cc @bdhirsh are the out variants *_copy ops generated for functionalization are properly tested via OpInfo?

Reply:
It looks like:

(1) most (but not all) {view}_copy ops have OpInfo tests today (for example, I don't see one for `split_copy`).

(2) Of those, most of them properly set `supports_out=True` for out= testing, but there are a few that do not (e.g. `diagonal_copy`) https://github.com/pytorch/pytorch/blob/main/torch/testing/_internal/common_methods_invocations.py#L11487

Reply:
OpInfo-based test added in https://github.com/pytorch/pytorch/pull/118512